tools/crm_node.c

```diff
@@ -153,9 +153,10 @@ int tools_remove_node_cache(const char *node, const char *target)
     }
 
     if(safe_str_eq(target, CRM_SYSTEM_CRMD)) {
-        admin_uuid = calloc(1, 11);
-        snprintf(admin_uuid, 10, "%d", getpid());
-        admin_uuid[10] = '\0';
+        admin_uuid = malloc(21);
```

Comment:
In all these cases, crm_strdup_printf() would be better as mentioned earlier.

But a general point about malloc() vs calloc(): in modern OSes including Linux, calloc() has no cost over malloc(), so we should prefer it for all our memory allocation, to have a known starting point and reduce the likelihood of future code changes overlooking the difference.